samples/demo-authorizationserver/src/main/java/sample/config/DefaultSecurityConfig.java

```diff
@@ -70,9 +79,9 @@ private AuthenticationSuccessHandler authenticationSuccessHandler() {
 	// @formatter:off
 	@Bean
 	public UserDetailsService users() {
-		UserDetails user = User.withDefaultPasswordEncoder()
-				.username("user1")
-				.password("password")
+        UserDetails user = User.builder()
+                .username("user1")
+                .password(bCryptPasswordEncoder().encode("password"))
```

Comment:
@xxxjjhhh 

With this change, the `demo-client` application fails to login `user1`. The log message on server is:

```
2024-03-22T16:17:37.027-04:00  WARN 72651 --- [nio-9000-exec-5] o.s.s.c.bcrypt.BCryptPasswordEncoder     : Encoded password does not look like BCrypt
```

Furthermore, I don't think this change is necessary as `User.withDefaultPasswordEncoder()` uses `BCryptPasswordEncoder` as the default.

I'm going to close this PR based on the explanation.